pep-9999.rst

```diff
@@ -0,0 +1,437 @@
+PEP: 9999
+Title: Override Decorator for Static Typing
+Author: Steven Troxler <steven.troxler at gmail.com>,
+        Joshua Xu <jxu425 at fb.com>,
+        Shannon Zhu <szhu at fb.com>
+Status: Draft
+Sponsor: FILLME
```

Comment:
You can put me (or Guido if he's interested :) ).

Reply:
Thanks!